yt_dlp/extractor/radiko.py

```diff
@@ -159,6 +162,9 @@ def _extract_formats(self, video_id, station, is_onair, ft, cursor, auth_token,
 
         return formats
 
+    def _extract_performers(self, prog):
+        return [x.strip() for x in try_call(lambda: re.split(self._PERFORMER_SPLIT_RE, prog.find('pfm').text))]
```

Comment:
This is still fatal (`try_call` returns `None` when it fails, which is not iterable), and the `artists` PR has not been merged yet, so we'll still need a string for now:
```suggestion
    def _extract_performers(self, prog):
        performers = traverse_obj(prog, (
            'pfm/text()', ..., {lambda x: re.split(r'[/／、　,，]', x)}, ..., {str.strip}))
        # TODO: change 'artist' fields to 'artists' and return traversal list instead of str
        return ', '.join(performers) or None
```